src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,29 @@
+
+@mixin clock-dial($size, $rim-bg, $bg) {
+  height: $size;
+  width: $size;
+  border: 8px solid $rim-bg;
+  background: $bg;
```

Comment:
Since you've used this mixin only once, you can remove border and background from it, and add border-radius: 50% (you have two circles in your layout, so better let's make this mixin reusable)